diff --git a/src/CVTLike/CVT.cpp b/src/CVTLike/CVT.cpp
index a43488e..6ef7910 100644
--- a/src/CVTLike/CVT.cpp
+++ b/src/CVTLike/CVT.cpp
@@ -45,7 +45,7 @@ typedef CGAL::AABB_face_graph_triangle_primitive<Polyhedron> Primitive;
 typedef CGAL::AABB_traits<K_T, Primitive> Traits;
 typedef CGAL::AABB_tree<Traits> Tree;
 typedef Tree::Point_and_primitive_id Point_and_primitive_id;
-double gamma = 0.00000000000001;
+double point_epsilon = 0.00000000000001;
 struct MyPoint
 {
 	MyPoint(Eigen::Vector3d a)
@@ -68,7 +68,7 @@ struct MyPoint
 
 
 		double dis = (p - a.p).norm();
-		if (dis < gamma)
+		if (dis < point_epsilon)
 		{
 			return false;
 		}
@@ -234,7 +234,7 @@ struct MyPoint
 
 		if (step >= 2)
 		{
-			std::string filepath = outpath + "\\\\Ours_" + std::to_string(num) + "_" + modelname + "_Remesh.obj";
+			std::string filepath = outpath + "Ours_" + std::to_string(num) + "_" + modelname + "_Remesh.obj";
 
 
 			std::string	filepath1 = outpath + "Ours_" + std::to_string(num) + "_" + modelname + "Iter" + std::to_string(step - 3) + "_Remesh.obj";
